scripts/add_rtp_task_jobid.py

```diff
@@ -14,59 +14,101 @@
 
 """
 
+import h5py
+import numpy as np
 from astropy.time import Time
 
 import hera_mc.mc as mc
 
 
+def get_obsid_from_file(filename):
```

Comment:
This logic looks duplicated from `add_rtp_process_event`. Maybe it should go in the utilities?

Reply:
Okay, I can break it out and put it into the utilities.

Reply:
Done in 1bb677e